src/util/message.h

```diff
@@ -42,8 +42,10 @@ struct messaget
   static inline class
   {
     template <typename... Args>
-    void println(FILE *f, Args &&...args) const
+    void println(FILE *f, VerbosityLevel lvl, Args &&...args) const
     {
+      if(lvl == VerbosityLevel::Error)
+        fmt::print(f, "ERROR: ");
```

Comment:
Sorry, I missed this part. We decided not to do it precisely because of this weird `ERROR: PARSING ERROR` kind of messages in https://github.com/esbmc/esbmc/pull/796#issuecomment-1183143762. This needs some uniformity.